src/lib/config/config.go

```diff
@@ -150,3 +163,22 @@ func GetNewRouteType(method, url string, headers []Headers, params []Params, bod
 		Labels: labels,
 	}
 }
+
+func LoadENV() {
+	err := godotenv.Load()
+	if err != nil {
+		panic(err)
+	}
+	PORT = os.Getenv("PORT")
+	if PORT == "" {
+		PORT = "9990"
+	}
+	SYSTEM_METRICS_PATH = os.Getenv("SYSTEM_METRICS_PATH")
+	JOURNAL_METRICS_PATH = os.Getenv("JOURNAL_METRICS_PATH")
+	CONFIGURATION_FILE_PATH = os.Getenv("CONFIGURATION_FILE_PATH")
+	STORAGE_DIR = os.Getenv("STORAGE_DIR")
+	PATH_PING = os.Getenv("PATH_PING")
+	PATH_JITTER = os.Getenv("PATH_JITTER")
+	PATH_FLOOD_PING = os.Getenv("PATH_FLOOD_PING")
+	PATH_MONITORING = os.Getenv("PATH_MONITORING")
```

Comment:
The variables in `src/main.go` had default values.
Should we add those default values here?

Reply:
According to me Default value will only exists for PORT rest has to be absolute Path. So It is must to have all the paths defined.
Correct me if I'm wrong.
Thanks.